test/decorator-annotator_test.ts

```diff
@@ -54,6 +55,21 @@ describe(
         expect(() => convertDecorators(program.getTypeChecker(), badSourceFile)).to.throw();
       });
 
+      it('generates a source map', () => {
+        const {output, sourceMap} = translate(`
+/** @Annotation */ let Test1: Function;
+@Test1
+export class Foo {
+}
+let X = 'a string';`);
+        const rawMap = (sourceMap as any).toJSON();
```

Comment:
It doesn't appear to be.  I copied and pasted from the source map test, I fixed it there as well.